src/sidebar/components/SearchInput.js

```diff
@@ -28,7 +85,10 @@ import { useSidebarStore } from '../store';
 export default function SearchInput({ alwaysExpanded, query, onSearch }) {
   const store = useSidebarStore();
   const isLoading = store.isLoading();
-  const input = /** @type {{ current: HTMLInputElement }} */ (useRef());
+  const input = /** @type {import('preact').RefObject<HTMLInputElement>} */ (
```

Comment:
This can be shortened a bit with the `const input = useRef(/** @type {HTMLInputElement|null} */ (null));` pattern that we use in various places.